src/apple/AppleMain.cxx

```diff
@@ -32,7 +32,9 @@ int apple_main(int argc, char *argv[])
 			_exit(0);
 		}
 	}
-#endif
 
+	MainConfigured(options, raw_config);
```

Comment:
Return value missing - does this even compile?

Reply:
It does, check the tasks